tests/vmi_lifecycle_test.go

```diff
@@ -465,7 +466,11 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			})
 
 			It("should be stopped and have Failed phase when a PanicDevice is provided", func() {
-				vmi := libvmifact.NewFedora(libvmi.WithPanicDevice(v1.Isa))
+				device := v1.Isa
```

Comment:
nit: I would do a table test out this

Reply:
What about providing some new `requires` decorators we can use in situations like this?

```go
DescribeTable("should be stopped and have Failed phase when a PanicDevice is provided", func(arch string, panicDevice v1.PanicDevice){
[..]
}
  Entry("with amd64 VMI", "amd64", v1.Isa, decorators.requiresAMD64WorkerNodes),
  Entry("with s390x VMI", "s390x", v1.Isa, decorators.requiresS390xWorkerNodes),
  Entry("with arm64 VMI", "arm64", v1.Pvpanic, decorators.requiresARM64WorkerNodes),
)
```